lib/console.js

```diff
@@ -162,6 +162,11 @@ Console.prototype.dir = function dir(object, options) {
 };
 
 
+Console.prototype.dirxml = function dirxml(...data) {
```

Comment:
Nit: This can be simplified to `Console.prototype.dirxml = Console.prototype.log` as far as I can tell.